bin/prom2teams

```diff
@@ -14,8 +14,10 @@ if __name__ == "__main__":
                                      'and sends it to Microsoft Teams using configured connectors ')
 
     parser.add_argument('-c', '--configpath', help='config INI file path', required=True)
+    parser.add_argument('-l', '--logfilepath', help='log file path', required=True)
```

Comment:
These two must be required?

Reply:
Yes, after this PR must be.

What should do if these arguments aren't provided? Should redirect messages to stdout/stderr? Should disable logging?  :)

Reply:
fixed 👍